ci/Jenkinsfile.tests

```diff
@@ -30,6 +30,8 @@ pipeline {
 
   environment {
     TARGET  = 'tests'
+    DB_CONT  = "status-go-test-db-${env.EXECUTOR_NUMBER.toInteger() + 1}"
```

Comment:
Why is this calling the container `status-go-test-db-...`? Pretty sure this isn't `status-go` repo.